sag_ji3.par

```diff
@@ -1,118 +1,118 @@
-64.161280243	(|\	8505/8192	m	33.2
-66.112954167	)|\\	885735/851968	�	34
-68.06462809
+-68.084530821955000	)!//	851968/885735		-34
+-66.117175958162600	(!/	8192/8505	d	-33.2
+-64.149821094370200	!//	360448/373977		-33
+-63.728245052129000	'(!/	27/28	'd	-32.4
+-62.465465808856200	.(!)	23068672/23914845	.w	-32
+-61.760890188336700	(!)	704/729	w	-31
+-59.793535324544400	!/)	8388608/8680203		-30.4
+-58.589460280797900	'(!)	495/512	'w	-30
+-57.826180460752100	)\!/	392/405		-29
+-55.858825596959800	.\!/	1048576/1082565	.v	-28.6
+-55.095545776914000	(\!	3969/4096		-28
+-53.891470733167500	\!/	32/33	v	-27
+-51.924115869375200	'\!/	10935/11264	'v	-26.6
+-51.219540248855700	.\!)	57344/59049	.&	-26
+-49.956761005582900	(!~	171/176		-25.8
+-49.535184963341700	\!)	35/36	&	-25
+-47.567830099549400	)\\!	405/416		-24
+-45.600475235757100	.\\!	38/39	._	-23
+-44.195221761619800	\\!	6400/6561	_	-22
+-41.665765508172500	'\\!	125/128	'_	-21
+-40.260512034035200	~!/	16384/16767		-20.8
+-39.698410644380200	(!(	44/45	a	-20
+-38.061940349778500	\!~	45/46		-19.9
+-37.871581128001700	.~!)	524288/535815		-19.6
+-37.309479738346700	'(!(	8019/8192	'a	-19
+-36.588490003137900	~!)	48/49		-18.4
+-35.118091464366500	.(!	49/50	.j	-18
+-34.358447442658300	(!	45056/45927	j	-17
+-32.391092578866000	!/	54/55	k	-16.6
+-31.407415146969900	'(!	55/56	'j	-16.4
+-30.985839104728600	)!)	56/57		-16
+-30.002161672832500	.!)	3584/3645	.t	-15
+-28.596908198695100	!)	63/64	t	-14
+-26.067451945247900	'!)	2066715/2097152	't	-13.4
+-24.885981585695500	)\!	40960/41553		-13
+-23.955448448755500	.\!	524288/531441	.\	-12.4
+-23.116419649559500	')\!	75/76	'	-12
+-22.694843607318300	\!	80/81	\	-11
+-20.305912701284800	)!~	19456/19683		-10.7
+-19.743811311629800	'\!	2025/2048	'\	-10
+-18.760133879733700	~~!	98/99		-9
+-17.354880405596300	!~	729/736	z	-8
+-15.387525541804000	~!(	4096/4131	o	-7
+-13.420170678011700	'~!(	135/136	'o	-6.6
+-12.436493246115600	)~!	143/144		-6.4
+-12.014917203874400	.)!(	2816/2835	.i	-6
+-11.031239771978200	)!(	891/896	i	-5
+-9.063884908185930	~!	2176/2187	s	-4.6
+-8.080207476289780	.!(	224/225	.n	-4.2
+-7.096530044393640	'~!	255/256	's	-4
+-6.534428654738690	!(	5103/5120	n	-3
+-4.567073790946400	)!	512/513	;	-2
+-2.740244274567840	.!	32768/32805	.h	-1.3
+-1.756566842671690	')!	1215/1216	';	-1
+-1.194465453016750
+1.194465453016750	.)|	1216/1215	.r	1
+1.756566842671690	'|	32805/32768	'	1.3
+2.740244274567840	)|	513/512	r	2
+4.567073790946400	|(	5120/5103	u	3
+6.534428654738690	.~|	256/255	.$	4
+7.096530044393640	'|(	225/224	'u	4.2
+8.080207476289780	~|	2187/2176	$	4.6
+9.063884908185930	)|(	896/891	*	5
+11.031239771978200	')|(	2835/2816	'*	6
+12.014917203874400	)~|	144/143		6.4
+12.436493246115600	.~|(	136/135	.e	6.6
+13.420170678011700	~|(	4131/4096	e	7
+15.387525541804000	|~	736/729	~	8
+17.354880405596300	~~|	99/98		9
+18.760133879733700	./|	2048/2025	./	10
+19.743811311629800	)|~	19683/19456		10.7
+20.305912701284800	/|	81/80	/	11
+22.694843607318300	.)/|	76/75		12

```

Comment:
interestingly, this was fine, though there was a mistake in the downwards section

Reply:
Well found.